README.md

```diff
@@ -39,6 +42,19 @@ nuget FsCheck 2.0.4
 nuget FsCheck 2.0.5-b247
 ```
 
+#### Using `FsCheck.NUnit` installed by Paket ####
+Note that [Paket doesn't run any powershell scripts](https://fsprojects.github.io/Paket/faq.html#Does-Paket-run-install-ps1-scripts).
+So if you want to use `FsCheck.NUnit` that you installed via Paket, you have to perform this step manually.
+Add reference to the following assemblies in the apropriate project(s):
+
+```
+packages/NUnit.Runners/tools/lib/nunit.core.dll
+packages/NUnit.Runners/tools/lib/nunit.core.interfaces.dll
+```
+
+Also make sure your project(s) has target `.NET 4.5` or greater.
+
+
```

Comment:
This doesn't belong in the readme, can you add it to https://fscheck.github.io/FsCheck/RunningTests.html "Using FsCheck.NUnit" paragraph instead?